controllers/nnf_workflow_controller.go

```diff
@@ -257,8 +257,7 @@ func (r *NnfWorkflowReconciler) handleUnsupportedState(ctx context.Context, work
 
 	log.Info("Unsupported Status.State", "state", workflow.Status.State)
 
-	// TODO: This should return an error, but for now unhandled states simply succeed to allow Flux to exercise workflow...
-	// Complete state in the drivers
+	// TODO: This should return an error, but for now unhandled states simply succeed to allow Flux to exercise worworkflow.Spec.DWDirectives[dwIndex]	// Complete state in the drivers
```

Comment:
Does this build? Where is worworkflow defined?
I think we handle all states now, so the comment is defunct.

Reply:
I'll leave the comment for now. We need to remove the code that was treating unhandled states as success. I'll save that for a different PR.